### ISSUE TITLE
Baritone mines its own trail that it built when trying to clear a vertical area

### ISSUE THREAD

**Issue by NOisgodly** [cabaletta/baritone#4569]
Baritone mines its own trail that it built when trying to clear a vertical area

see the provided images
![image](https://github.com/user-attachments/assets/3d244595-bb83-4529-b240-b5967f3b5354)
![image](https://github.com/user-attachments/assets/8a04a874-c42e-4100-8b9f-55072bb73775)
![image](https://github.com/user-attachments/assets/e770312f-f1f5-485b-8efd-223d52d220be)

its mining its own bridge staircase thing while attempting to mine an area making it unable to mine an area.

## Final checklist
- [✔] I know how to properly use checkboxes 
- [✔ ] I have not used any OwO's or UwU's in this issue.

**Comment by ZacSharp** [comment]
Yeah, known issue but super hard to fix properly. To work around it you can turn on `buildInLayers` (and toggle `layerOrder` if it starts from the wrong side)
